ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis.ecf

```diff
@@ -29,14 +21,14 @@ export model=gfs
 ############################################################
 # Load modules
 ############################################################
-module load lsf/${lsf_ver}
-module load impi/${impi_ver}
-module load CFP/${CFP_ver}
-module load python/${python_ver}
-module load NetCDF-parallel/${netcdf_parallel_ver}
-module load HDF5-parallel/${hdf5_parallel_ver}
+module load netcdf/${netcdf_ver}
```

Comment:
Where are the PrgIntel, intel, mpich, cray-pals etc modules loaded?
I thought they are needed to execute.

I see in jgdas_atmos_dump.ecf, some of these are loaded.
So I wonder why the inconsistency?